### DISCUSSION TITLE
Problem during installation of BthPS3 on Windows 11

### DISCUSSION THREAD

**Discussion by Necrow51** [nefarius/BthPS3#39] [Support]
Hello, I have this issue, I can't install :(
ScpToolkit work a charm but the wireless randomly no longer works after computer reboot with my Dualshock 3
![Sans titre 2](https://user-images.githubusercontent.com/88400526/150648903-d9d3c1aa-5d76-4f95-95da-74d5fbfde2a0.jpg)

**Comment by nefarius** [comment]
Works perfectly over here:

![image](https://user-images.githubusercontent.com/286631/150649366-f6a0fc5e-4bba-4c43-927f-b5a7a758ff32.png)

Do you have this registry value on your machine?

![image](https://user-images.githubusercontent.com/286631/150649380-2e3ad15a-1859-4bdc-a2c1-60feb902861b.png)

**Reply by Necrow51** [reply]
Now it's installed thanks but when I unplug my DS3 controller, she is totally disconnected, bluetooth won't work :(
![Sans titre 2](https://user-images.githubusercontent.com/88400526/150649708-232ced89-316d-44d9-b67b-03a900da42dc.jpg)

**Reply by nefarius** [reply]
Shibari is long deprecated, [use DsHidMini](https://vigem.org/projects/DsHidMini/How-to-Install/) instead.

**Comment by Necrow51** [comment]
All it's OK now, thanks a lot
Issue fixed
